sound/soc/bcm/rpi-wm8804-soundcard.c

```diff
@@ -140,26 +174,45 @@ static int snd_rpi_wm8804_hw_params(struct snd_pcm_substream *substream,
 		break;
 	case 44100:
 		sampling_freq = 0x00;
+		gpiod_set_value_cansleep(led_gpio_1, 1);
```

Comment:
This code is shared with all the other wm8804-based cards. You should create your own hw_params function that calls this one and also sets the LEDs as you want them. You can hook that in by providing a .ops member to snd_interlude_audio_digital_dai.